cli_tools/gce_vm_image_import/importer/importer.go

```diff
@@ -58,15 +60,23 @@ type importer struct {
 	processorProvider processorProvider
 	traceLogs         []string
 	diskClient        diskClient
+	timeout           time.Duration
+	processor         processor
+	cancelReason      string
 }
 
-func (i importer) Run(ctx context.Context) (loggable service.Loggable, err error) {
-
+func (i *importer) Run(ctx context.Context) (loggable service.Loggable, err error) {
+	if i.timeout.Nanoseconds() > 0 {
+		var cancel func()
+		ctx, cancel = context.WithTimeout(ctx, i.timeout)
```

Comment:
For now don't cancel the context you pass to daisy, I have seen this cause behavior in cleanup that we would not want. Instead use the existing workflow cancel functions to tell a workflow to cancel and cleanup, you most likely will have to spin up a goroutine that just waits for a timer that then calls the cancel function

Reply:
Changed to pass a new context to Daisy and use importer context for importer timeout